src/telemetry/telemetryService.ts

```diff
@@ -130,6 +130,7 @@ export default class TelemetryService {
         this._context.extensionPath,
         './constants.json'
       );
+      // eslint-disable-next-line no-sync
       const constantsFile = fs.readFileSync(segmentKeyFileLocation).toString();
```

Comment:
might as well clean this up while we’re at it:

```suggestion
      const constantsFile = fs.readFileSync(segmentKeyFileLocation, 'utf8');
```